tests/test_date.py

```diff
@@ -726,6 +726,34 @@ def test_timestamp_in_microseconds(self):
             datetime.fromtimestamp(1570308760).replace(microsecond=263111)
         )
 
+    @parameterized.expand([
+        param(
+            input_timestamp='-1570308760',
+            negative_allowed=True,
```

Comment:
This seems to indicate that negative and positive are both parsed. Maybe:

```suggestion
            negative=True,
```